admin_manual/configuration/user/user_provisioning_api.rst

```diff
@@ -245,6 +245,100 @@ XML Output
     <data/>
   </ocs>
 
+Enable User
+-----------
+
+Enable a user.
```

Comment:
This line seems quite superfluous, given the title above. Can you remove it in favour of a slightly meatier, yet still succinct, description?